core/src/test/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParserTest.java

```diff
@@ -297,7 +292,7 @@ public void testFileOlderVersion() throws Exception {
     assertNotNull(configFile);
 
     Map<String,String> contents = ClusterConfigParser
-        .parseConfiguration(Path.of(configFile.toURI()).toFile().getAbsolutePath());
+        .parseConfiguration(Path.of(configFile.toURI()).toAbsolutePath().toString());
```

Comment:
ClusterConfigParser could take a Path object instead of a String. That would simplify this further, and make it more type-safe. ClusterConfigParser is only used internally.